src/app/components/work-item-iteration-modal/work-item-iteration-modal.component.ts

```diff
@@ -200,17 +207,28 @@ export class FabPlannerAssociateIterationModalComponent {
         this.selectedIteration = null;
         this.workItem.relationships.iteration = workItem.relationships.iteration;
         this.workItem.attributes['version'] = workItem.attributes['version'];
-        this.workItemService.emitEditWI(workItem);
+        this.workItemService.emitEditWI(this.workItem);
       });
   }
 
   open(event: any) {
     this.getIterations();
+    this.workItemPayload = {
+      id: this.workItem.id,
+      number: this.workItem.number,
```

Comment:
How do you get number is **this.workItem** ?

Reply:
Ah my bad will fix it.